components/collective-page/sections/Contribute.js

```diff
@@ -239,7 +239,8 @@ class SectionContribute extends React.PureComponent {
     const hasNoContributor = !this.hasContributors(contributors);
     const sortedTicketTiers = this.sortTicketTiers(this.filterTickets(tiers));
     const hideTicketsFromNonAdmins = (sortedTicketTiers.length === 0 || !collective.isActive) && !isAdmin;
-    const cannotOrderTickets = (!hasContribute && !isAdmin) || (!canOrderTicketsFromEvent(collective) && !isAdmin);
+    const cannotOrderTickets =
+      (!hasContribute && !isAdmin) || (!canOrderTicketsFromEvent(collective) && !isAdmin) || isPastEvent(collective);
```

Comment:
Can you look at the code in `canOrderTicketsFromEvent` and reconsider?

Reply:
Ops, yes, I've missed that fact we already check `isPastEvent` within the `canOrderTicketsFromEvent` function. So then all we got to do here is to remove the `!isAdmin` and we are good. I've made this change. Hope this simplifies things.

Reply:
I think we can simplify more here. Let me try some things first. 🤔

Reply:
I've some some simplifications so that we only show the `Tickets` section if the event has not passed/expired. This is true for admins and non-admins. Now I also realized even if we hide the Tickets section admins can go tot he settings and add/remove tickets. So I think there's no point to showing the Tickets section in the settings for past events with this change. I have made a change to make sure the tickets section (in settings) is not shown as well for past events. Let me know your thoughts. Thanks.